rocketpy/rocket/aero_surface.py

```diff
@@ -34,6 +34,30 @@ def __init__(self, name):
         self.name = name
         return None
 
+    # Defines beta parameter
+    def _beta(_, mach):
```

Comment:
This `_beta` method doesn't use any class attribute. It should be either a static method or a an isolated function commonly defined in tools.py. This would also avoid making the method private.

I don't really see this function being applied outside of the context of aero surfaces. Therefore, I would convert it into a `@staticmethod`

Reply:
https://github.com/RocketPy-Team/RocketPy/pull/609/commits/0b44194de7ad19d6f914ce84b0ef1a4e5c8c7ea5